templates/lean_projects.html

```diff
@@ -34,6 +34,11 @@ <h1>Lean projects</h1>
       please see the directions at the
       <a href="https://github.com/leanprover-contrib/leanprover-contrib">leanprover-contrib</a> page.
     </p>
+
+    <p>
+      This list and the repository that manages it are both works in progress.
+      Please add your own project and report any problems!
```

Comment:
Maybe link to the place(s) to report e.g. Zulip / github issues?

Otherwise LGTM, feel free to merge.